### ISSUE TITLE
window tab language is not in english

### ISSUE THREAD

**Issue by dajo09** [moneymanagerex/moneymanagerex#2370]
<img width="831" alt="Screen Shot 2020-04-08 at 11 38 38 PM" src="https://user-images.githubusercontent.com/33592524/78865783-200da380-79f3-11ea-87e7-4976d2f1e654.png">
I set the language to English. I have uninstalled and reinstalled it a couple of times. I have to run 'sudo xattr -r -d com.apple.quarantine/Applicatons/mmex.app' everytime I install it because it gives me this message: "mmex.app" is damaged and can't be opened. 

**Your OS:**
- [ ] Windows
- [x] Mac OSX
- [ ] Linux 
- [ ] Android -> please report bugs to moneymanagerex/android-money-manager-ex repo!

**MMEX version:**
- [ ] 1.4.x
- [x] 1.3.x
- [ ] 1.2.x
- [ ] 1.1 or older
- [ ] development version (git master branch)

**Comment by dajo09** [comment]
<img width="831" alt="Screen Shot 2020-04-08 at 11 38 38 PM" src="https://user-images.githubusercontent.com/33592524/78865783-200da380-79f3-11ea-87e7-4976d2f1e654.png">
I set the language to English. I have uninstalled and reinstalled it a couple of times. I have to run 'sudo xattr -r -d com.apple.quarantine/Applicatons/mmex.app' everytime I install it because it gives me this message: "mmex.app" is damaged and can't be opened.
